webpack.config.js

```diff
@@ -24,13 +24,34 @@ const config = {
 	},
 	resolve: {
 		extensions: ['.js', '.jsx', '.json'],
+		alias: {
+			'@': path.resolve(__dirname, 'src'),
+			'@pages': path.resolve(__dirname, 'src/pages'),
+			'@components': path.resolve(__dirname, 'src/components'),
+			'@utils': path.resolve(__dirname, 'src/utils'),
+			'@forms': path.resolve(__dirname, 'src/forms'),
+		},
+		modules: ['node_modules'],
```

Comment:
What's the use of these lines of code?

Reply:
### Aliases are a way to let web-pack know where to find our code by providing a word or character that  represents a partial reference to where the code is located.

- [x] as you see in resolve, we are telling the web-pack to look for listed  file extensions when compiling example
   `extensions: ['.js', '.jsx', '.json'],` .
I included these lines in my webpack.config because i though that later on I will be working with components, pages or forms, but the issue that I've made is to define it before. but in other side that's it.       
### Does it make sense?? 
#### if not [read more](https://dev.to/mmcshinsky/webpack-aliases-keep-my-code-sane-3c3e)

Reply:
That's great then good job!